04-onion-routing.md

```diff
@@ -575,7 +597,7 @@ HTLC at the final hop:
 2. data:
    * [`4`:`cltv_expiry`]
 
-If the `amt_to_forward` does not match the `incoming_htlc_amt` of
+If the `amt_to_forward` is higher than `incoming_htlc_amt` of
```

Comment:
Yes, you're right, my mistake!  To avoid further delay, I'll force pushed back the version prior to your restoration, then merge right now.